DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,121 @@
+# Developer Guide
+
+The developer guide is an on-boarding document for new contributors to FileOnQ.Imaging.Raw. This project implements some non-standard practices and the goal of this document is to answer questions and explain how the code works.
+
+## Compiling/Building
+
+TBD
```

Comment:
This should be very similar to what we wrote in heif as most of the setup instructions are in the `BUILDING.md` file